CuitService/CuitNumber.cs

```diff
@@ -35,7 +35,7 @@ public CuitNumber(string value)
             FormattedValue = CuitRegex.Replace(SimplifiedValue, "$1-$2-$3");
         }
 
-        public static ValidationResult ValidateNumber(string? value)
+        public static ValidationResult? ValidateNumber(string? value)
```

Comment:
In my opinion, this change is wrong, ValidationResult will never be null.

Reply:
In net5 ValidationResult.Success is the type ValidationResult_?_ in net core 3.1 is type ValidationResult

Reply:
![image](https://user-images.githubusercontent.com/6796523/114239269-3cbd1e00-995c-11eb-9168-ab832017d965.png)

Reply:
If I am not wrong this is an error or a pending thing in their backlog.

I think that we can confirm that `ValidationResult.Success` is not null and guarantee that for _ValidateNumber_ with this signature: `public static ValidationResult ValidateNumber(string? value)`

Reply:
Well, the problem is that they use the constant value _null_ in place of using the [_null object pattern_](https://en.wikipedia.org/wiki/Null_object_pattern), so, in this case, we cannot take advantage of _not nullable reference types_ :( 

> _The null value is used to indicate success. Consumers of ValidationResult should compare the values to Success rather than checking for null._
>
> [ValidationResult.Success Field's documentation remarks](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.validationresult.success?view=net-5.0#remarks)